eng/pipelines/templates/jobs/perf.yml

```diff
@@ -33,10 +33,10 @@ extends:
     InstallLanguageSteps:
     - task: JavaToolInstaller@0
       inputs:
-        versionSpec: '8'
+        versionSpec: '17'
```

Comment:
Could we make this something that is configurable in the pipeline with a default of Java 17?

Reply:
Good suggestion, I will do this for all languages.  I also need to make some changes to the perf automation.  I was originally planning to have the perf automation app control the language version (which works for .NET), but for other languages we usually configure the language version in the pipeline YML.  I will change the perf automation app to just **report** the language version used, not try to **select** the language version used.